tests/test_model.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+"""Tests for `genai_stack` package."""
+
+
+import unittest
+
+from genai_stack.model import OpenAIGpt35Model
+
+
+class TestModel(unittest.TestCase):
+    def test_openai_gpt35_model(self):
```

Comment:
Add testcases for other models also

Reply:
I don't think its required because running those testcases will take too much time because we need to download the models first & if there is any way were we can simulate the behavior then we can write it otherwise I don't think we should write such heavy tests

Reply:
ok got it